frontend/fingertips-frontend/components/molecules/Pill/index.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { Paragraph } from 'govuk-react';
+import { typography } from '@govuk-react/lib';
+import React from 'react';
+import styled from 'styled-components';
+
+interface PillProps {
+  selectedFilterName: string;
+}
+
+const StyledDivContainer = styled('div')({
+  backgroundColor: 'white',
+  border: '2px black solid',
+  padding: '10px 20px',
```

Comment:
Can we use `em` rather that `px` that will give us some scaling for different screen sizes. Also the padding looks bit too much. If you can scale it down little more please.